pkgs/applications/version-management/git-and-tools/git/default.nix

```diff
@@ -115,6 +117,10 @@ stdenv.mkDerivation {
   # See https://github.com/Homebrew/homebrew-core/commit/dfa3ccf1e7d3901e371b5140b935839ba9d8b706
   ++ lib.optional stdenv.isDarwin "TKFRAMEWORK=/nonexistent";
 
+  disallowedReferences = lib.optionals (stdenv.buildPlatform != stdenv.hostPlatform) [
```

Comment:
This is causing rebuilds

Reply:
The change to the makeFlags as well, I think.

Reply:
Thanks, that's the reason. I will update this PR tomorrow.

Reply:
It s totally fine if this goes to staging. We don't need to try to reduce rebuilds if the solution is working.



----


@ofborg eval